modules/@apostrophecms/page/lib/legacy-migrations.js

```diff
@@ -100,62 +99,6 @@ module.exports = (self) => {
           }
         });
       });
-    },
-    addMissingLastTargetIdAndPositionMigration() {
```

Comment:
We don't store these anymore, but this code became the basis of inferring them.

Reply:
As a general rule we avoid removing data because we are fallible, make mistakes, and might regret not having it later 😄 I think that applies here.